.github/workflows/pullpreview.yml

```diff
@@ -96,6 +104,10 @@ jobs:
           instance_type: ${{ inputs.INSTANCE_TYPE }}
           # Ports to open on the server
           ports: 80, 443
+          # Use a custom domain for the preview environment
+          dns: ${{ inputs.DNS }}
```

Comment:
what happens if the input is empty?

Reply:
PullPreview leaves the domain empty. The url therefore only consists of the protocol and the subdomain. 

Therefore there is now an fallback to the default PullPreview url, when the input is empty.